dockerfile

```diff
@@ -1,7 +1,7 @@
 FROM python:3.11-slim
 
 RUN apt update && apt upgrade -y
-COPY frontend/requirements.txt /requirements.txt
+COPY frontend/requirements.in /requirements.txt
```

Comment:
I think the current method is fine -- the key requirements are defined in the requirements.in file, which is then compiled to a full set of requirements using "pip-compile".  See the included "upgrade-deps.sh" file in the repo  -- you will need to pip-install "pip-tools" to run it.

Reply:
But is there a file /frontend/requirements.txt?

Reply:
This script creates the requirement.txt files. You may choose to comment out the pip-sync line if you do not wish to install the Python dependencies outside of Docker.

![Screenshot 2024-05-22 193402](https://github.com/cam-digital-hospitals/frontend/assets/7994343/2f66441c-a72a-4800-9e2d-ed05b4c9ff7a)